src/Lorentz/Contracts/Metadata.hs

```diff
@@ -14,7 +14,35 @@ module Lorentz.Contracts.Metadata
 
 import Lorentz
 
-import Lorentz.Contracts.Spec.FA2Interface (TokenId, TokenMetadata(..), theTokenId)
+import Fmt (Buildable(..), (+|), (|+))
+
+import Lorentz.Contracts.Spec.FA2Interface (TokenId, theTokenId)
+
+data TokenMetadata = TokenMetadata
```

Comment:
Is there a specific reason to not use the new `TokenMetadata` from `FA2Interface`?

AFAIK we'd just need our own `mkTokenMetadata` that takes more arguments, but the `Map` could be used in place of the custom type.

Reply:
Yes, `TokenMetadata` in FA2 has changed. However, we have to stay backward-compatible

Reply:
> but the Map could be used in place of the custom type.

Yes, but `tzbtc-client` won't be able to work with a contract that is already deployed on mainnet

Reply:
Ok that makes perfect sense :ok_hand: